### DISCUSSION TITLE
Connect from phone to esp32 and after A2DPsource

### DISCUSSION THREAD

**Discussion by dandandan1983** [pschatzmann/ESP32-A2DP#473] [Q&A] [answered]
Hello. Maybe it's a strange question because i don't understand something in your library.

I use ESP32 WROOM-32. I want to read data from microphone -> A2DP Source -> Mobil Phone.

I catch issue. I can't connect from phone to ESP32. My phone can't see ESP32 Bluetooth module. 
If i used A2DP Sink, i can connect to bluetooth by Phone but i can't write sounds to phone. Or i don't know how to do it. 
Is it possible to start connection from Phone, not from Bluetooth module then i use A2DP Source ?

For example. https://github.com/pschatzmann/ESP32-A2DP/blob/main/examples/bt_music_sender/bt_music_sender.ino
My phone didn't see this bluetooth point: _LEXON MINO L_.
`a2dp_source.start("LEXON MINO L", get_data_frames); `
I want to have ability for connection from the Phone if it possible. I want to see a point from Phone, connect to it and start receiving audio data.
Is it possible? Thank you for your library.

p.s.
If i change it like my phone name, nothing happens.

**Comment by pschatzmann** [comment, marked as answer]
This has nothing to do with my library.
As far as I now, mobile phones can not be used as A2DP Sink (Bluetooth Speaker)!
Or have you ever been able to send Audio from one phone to another via A2DP ?

**Reply by dandandan1983** [reply]
Before your unswer i thought that all Phones use A2DP for that. I thought that i can do my own system like headphones.
Thank you for your answer.
